Marlin/example_configurations/Anet/A8/Configuration.h

```diff
@@ -425,7 +426,7 @@
 // or to allow moving the extruder regardless of the hotend temperature.
 // *** IT IS HIGHLY RECOMMENDED TO LEAVE THIS OPTION ENABLED! ***
 #define PREVENT_COLD_EXTRUSION
-#define EXTRUDE_MINTEMP 170
+#define EXTRUDE_MINTEMP 160  // 160 guards against false tripping when the extruder fan kicks on.
```

Comment:
If it reads 160 then your hotend is actually getting cold. Wrap your heater block in ceramic tape if the fan cools it too much.